aiogram/api/types/user.py

```diff
@@ -33,6 +41,80 @@ class User(TelegramObject):
 
     @property
     def full_name(self) -> str:
+        """
+        Get full name of user.
+        :return: str
+        """
         if self.last_name:
             return f"{self.first_name} {self.last_name}"
         return self.first_name
+
+    @property
+    def mention(self) -> str:
+        """
+        Get user's username to mention him.
+        Full name will be returned if user has no username.
+        :return: str
+        """
+        if self.username:
+            return "@" + self.username
+        return self.full_name
```

Comment:
How come? Do we have the same in 2.x? I think, method has very unclear behaviour

Reply:
Not only that, it also does not do what is expected of it when the `User` does not have a `username`. I mean, displaying a `full_name` is not a mention.

Reply:
we don't want to repeat our mistakes :)
I think we can remove it.